olad/www/console_values.html

```diff
@@ -1,728 +1,746 @@
 <html>

```

Comment:
@nomis52 should this file even be in OLA, it seems to be a test one that snuck in ages ago.